test/image-blueprints/layer1-base/group2/rhel94-brew-y2-with-optionals.toml

```diff
@@ -0,0 +1,74 @@
+{{- if env.Getenv "BREW_Y2_RELEASE_VERSION" "" -}}
+{{- /*
+
+  We wrap this template in a test so that the body of the output is
+  empty when there is no "current" version release. The output file
+  must end up completely empty, so we need to remove whitespace from
+  around the first and last template instructions.
+
+*/ -}}
+
+name = "rhel-9.4-microshift-brew-optionals-{{ .Env.BREW_Y2_RELEASE_VERSION }}"
```

Comment:
Why do we need to use variable names here? It would be rather hard to reference the image.

Can't we use `rhel-9.4-microshift-brew-y2-with-optionals` like we do for containers?